app/views/users/_sidebar.html.erb

```diff
@@ -32,52 +32,68 @@
   <% end %>
 
   <% cache "user-profile-sidebar-profile-details-#{@user.id}-#{@user.profile_updated_at}", expires_in: 10.days do %>
-    <% if @user.mostly_work_with.present? %>
-      <div class="crayons-card crayons-card--secondary">
-        <header class="crayons-card__header">
-          <h3 class="crayons-subtitle-3">Skills/languages</h3>
-        </header>
+    <% if FeatureFlag.enabled?(:profile_admin) %>
```

Comment:
This should be safe to deploy to DEV because all of the UI changes are behind feature flags.